operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -347,13 +348,32 @@ type ManagementApiAuthConfig struct {
 }
 
 type ReaperConfig struct {
+	// Other properties in ReaperConfig are ignores unless this is set to true. When set to
+	// true, the operator will enable remote JMX in the Cassandra cluster since Reaper
+	// manages repairs via JMX. Note that setting this to true after the Cassandra cluster
```

Comment:
nit - is this more clear about the expected behavior?
```suggestion
	// manages repairs via JMX. Note that toggling this to true after the Cassandra cluster
```